backend/PyMatcha/utils/populate_database.py

```diff
@@ -80,7 +85,8 @@ def populate_users(amount=150, drop_user_table=False):
 
             for tag in tags:
                 Tag.create(name=tag, user_id=u.id)
-            Image.create(user_id=u.id, link=user.get_picture(), is_primary=True)
+            image_url = requests.get(f"https://picsum.photos/seed/{username}/1000/1000").url
```

Comment:
The https://picsum.photos/seed/{username}/1000/1000 also could be a constant somewhere for improved readability